src/pyhf/writexml.py

```diff
@@ -48,11 +46,9 @@ def _make_hist_name(channel, sample, modifier='', prefix='hist', suffix=''):
 
 
 def _export_root_histogram(histname, data):
-    hist = TH1.from_numpy((np.asarray(data), np.arange(len(data) + 1)))
-    hist._fName = histname
```

Comment:
@jpivarski I lied and have _another_ follow up question. Is it possible to set a `name` attribute if creating a `TH1D` object from an `uproot.Model` with some later call?

```python
>>> import uproot
>>> import numpy as np
>>> data = np.arange(10)
>>> edges = np.arange(len(data) + 1)
>>> hist_model = uproot.to_writable((np.asarray(data), np.arange(len(data) + 1)))
>>> hist_model.name  # not set
>>> hist_model.name = "test"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: can't set attribute
```

Looking at the `help(hist_model)` show that 

```
class Model_TH1D_v3(uproot.behaviors.TH1.TH1, uproot.model.VersionedModel)
...
 |  ----------------------------------------------------------------------
 |  Readonly properties inherited from uproot.behaviors.TH1.Histogram:
 |  
 |  name
 |      The name of the histogram.
 |  
 |  title
 |      The title of the histogram.

```

Is there anyway to set this using something in the non-public API lke we were here with the `_fName` attribute with `uproot3_methods.classes.TH1`?

Reply:
Ah. Yes:

```python
>>> import uproot
>>> import numpy as np
>>> data = np.arange(10)
>>> edges = np.arange(len(data) + 1)
>>> hist_model = uproot.to_writable((np.asarray(data), np.arange(len(data) + 1)))
>>> hist_model.name  # empty here
>>> hist_model._members["fName"] = "test"
>>> hist_model.name
'test'
```

Reply:
...and I now see that none of this really mattered as `_ROOT_DATA_FILE[histname] = hist_uproot_model` will set the `_ROOT_DATA_FILE[histname].name` to `histname`.